src/piano_metrics/key_detection_test.py

```diff
@@ -0,0 +1,240 @@
+from typing import List
+
+import pytest
+import numpy as np
+import pandas as pd
+
+from src.piano_metrics.key_distribution import SpiralArray, analyze_piece, detect_key_from_notes
```

Comment:
```suggestion
from piano_metrics.key_distribution import SpiralArray, analyze_piece, detect_key_from_notes
```

You shouldn't have to import modules using absolute paths, but rather submodules. The package is defines in pyproject.yaml, and `piano_metrics` should be available after doing `pip install -e .`